commands/games/memory.py

```
@@ -0,0 +1,356 @@
+        game: MemoryGame,
+        command_info: utility.CommandInfo,
+    ) -> None:
+        super().__init__(timeout=300)
+        self.game = game
+        self.command_info = command_info
+        self._build_buttons()
+
+    def _build_buttons(self) -> None:
+        """Create buttons for the current game state."""
+        self.clear_items()
+        available = self.game.get_button_labels()
```

Flake8 (7.3.0)

[error] local variable 'available' is assigned to but never used

(F841)